### PR TITLE
use a computed instead of prototype extensions

### PR DESCRIPTION
**Pull request by efx**
Addresses #2 .

### CODE REVIEW

**Comment by efx** [comment]
@bguiz thoughts on this change?

File: trees/app/components/json-pretty.js [hunk 1]
```diff
@@ -20,12 +20,12 @@ var JsonPrettyComponent = Ember.Component.extend({
...
         }
         return new Ember.String.htmlSafe(out);
-    }.property('obj'),
+    }),
 
     //Thanks to: http://jsfiddle.net/KJQ9K/
     highlightSyntax: function(json) {
         json = json.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
-        return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?)/g, function (match) {
+        return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?)/g, function(match) {
```
Comment (bguiz) [inline trees/app/components/json-pretty.js]: Minor: This diff line seems extraneous, looks like it just removes a space character. Please revert this change.

Comment (efx) [reply trees/app/components/json-pretty.js]: Good catch. My default linting config creeping in :)

**Comment by bguiz** [comment]
@efx thanks for the patch, I've merged and published as `v0.1.3` https://yarnpkg.com/en/package/ember-cli-json-pretty

**Comment by efx** [comment]
sure thing @bguiz! excellent, now we can point at your package.
